frontend/lib/resolution/scope-queries.cpp

```diff
@@ -3463,7 +3543,7 @@ emitMultipleDefinedSymbolErrorsQuery(Context* context, const Scope* scope) {
                         LOOKUP_IMPORT_AND_USE |
                         LOOKUP_SKIP_SHADOW_SCOPES;
   for (auto name : namesDefinedMultiply) {
-    std::vector<BorrowedIdsWithName> v =
+    MatchingIdsWithName v =
```

Comment:
Consider `auto` in cases like these

Reply:
It's nice to put the type so less needs to be consulted to see what the code is doing.